src/Dropdown.php

```diff
@@ -181,10 +205,15 @@ public static function show($itemtype, $options = [])
         }
 
         if (!$item instanceof CommonTreeDropdown) {
-            $name = Sanitizer::unsanitize($name);
+            if ($params['multiple']) {
+                array_map(function ($name_to_unsanitize) {
+                    return Sanitizer::unsanitize($name_to_unsanitize);
+                }, $names);
```

Comment:
This method already handle arrays.
```suggestion
                $names = Sanitizer::unsanitize($names);
```